definitions/tools/star_align.cwl

```diff
@@ -0,0 +1,208 @@
+      otherwise no alignments will be output
+
+'
+    sjdbOverhang:
+        type: string?
+        inputBinding:
+            position: 14
+            prefix: '--sjdbOverhang'
+        doc: '
+
+      int>0: length of the donor/acceptor sequence on each side of the junctions,
+      ideally = (mate_length - 1)
+
+'
+    sjdbGTFfile:
+        type: File?
+        inputBinding:
+            position: 15
+            prefix: '--sjdbGTFfile'
+        doc: |
+          string: path to the GTF file with annotations
+    outFileNamePrefix:
+        type: string
+        inputBinding:
+            position: 16
+            prefix: '--outFileNamePrefix'
+        doc: |
+          string: output files name prefix (including full or relative path). Can
+          only be defined on the command line.
+    readFilesCommand:
+        type: string?
+        inputBinding:
+            position: 17
+            prefix: '--readFilesCommand'
+        doc: |
+          string(s): command line to execute for each of the input file. This command should generate FASTA or FASTQ text and send it to stdout
+          For example: zcat - to uncompress .gz files, bzcat - to uncompress .bz2 files, etc.
+    outSAMattributes:
+        type: string[]
+        default: ["NH", "HI", "AS", "NM", "MD"]
+        inputBinding:
+            position: 18
+            prefix: '--outSAMattributes'
+        doc: |
+          Standard
+          string: a string of desired SAM attributes, in the order desired for the output SAM
+          NH HI AS nM NM MD jM jI XS ... any combination in any order
+          Standard   ... NH HI AS nM
+          All        ... NH HI AS nM NM MD jM jI
+          None ... no attributes
+    outSAMattrRGline:
+        type:
+            type: array
+            items: string
+        inputBinding:
+            position: 19
+            prefix: '--outSAMattrRGline'
+        doc: |
+           -
+           string(s): SAM/BAM read group line. The first word contains the read group identifier and must start with "ID:", e.g. --outSAMattrRGline ID:xxx CN:yy "DS:z z z".
+           xxx will be added as RG tag to each output alignment. Any spaces in the tag values have to be double quoted.
+           Comma separated RG lines correspons to different (comma separated) input files in --readFilesIn. Commas have to be surrounded by spaces, e.g.
+           --outSAMattrRGline ID:xxx , ID:zzz "DS:z z" , ID:yyy DS:yyyy
+
+
+outputs:
+    aligned_bam:
+        type: File
+        outputBinding:
+          glob: "$(inputs.outFileNamePrefix)Aligned.sortedByCoord.out.bam"
+          #glob: "aligned.bam"
+    log_final:
+        type: File
+        outputBinding:
+          glob: "$(inputs.outFileNamePrefix)Log.final.out"
+    log:
+        type: File
+        outputBinding:
+          glob: "$(inputs.outFileNamePrefix)Log.out"
+    log_progress:
+        type: File
+        outputBinding:
+          glob: "$(inputs.outFileNamePrefix)Log.progress.out"
```

Comment:
what are these logs? Are they useful above and beyond what gets output to stdout/stderr and captured by cromwell?  Is there a reason to expose them as an output (as opposed to just leaving them in the execution directory for debugging?)

Reply:
IIRC these logs are good files that can be fed to multiqc.